osu.Game/Skinning/SkinResourceStore.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using System.IO;
+using System.Linq;
+using osu.Framework.IO.Stores;
+
+namespace osu.Game.Skinning
+{
+    public class SkinResourceStore : IResourceStore<byte[]>

```

Comment:
This should be called something more like `LegacySkinResourceStore` and maybe private to `LegacySkin`. We definitely want namespacing in the future.

Reply:
sounds good